### DISCUSSION TITLE
One-off invocation of Cloudprober?

### DISCUSSION THREAD

**Discussion by FranklinYu** [cloudprober/cloudprober#415] [Q&A] [answered]
I would like to export some disk information (S.M.A.R.T.) of my machines. In contrast to CPU and RAM, disk information rarely changes, so it makes little sense to keep a process running; I would rather set up Systemd timer to log the number daily, and upload to Stack Driver (this is the only backend I need). Does Cloudprober provide such a mode of operation? If not, would it make sense for me to reuse part of the Cloudprober codebase (such as the surfacer implementation)?

**Comment by manugarg** [comment, marked as answer]
@FranklinYu Sorry, I missed this thread. You can simply start the process with the probe interval set to say 5 min and kill it after 7 minutes or so.

I don't recommend using surfacers package directly as interface may change over time.
